xAPI.md

```diff
@@ -2743,28 +2667,29 @@ string form.  It is recommended variant 2 in [RFC 4122](http://tools.ietf.org/ht
 
 ### 4.5 ISO 8601 Timestamps
 
-Timestamps are a format type (and also a Statement property of type Timestamp) which are strings which represent 
-a specific time.  They are formatted according to ISO 8601's normal format.  Statements sent to an LRS can be 
-expected (a MUST requirement on the LRS) to keep precision to at least milliseconds (3 decimal points beyond 
-seconds).  
+Timestamps are a format type which represent a specific time.  They are formatted according to ISO 8601's normal format. 
+Statements sent to an LRS can be expected to keep precision to at least milliseconds 
 
 ###### Requirements
-* A timestamp MUST be formatted according to [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations).
-* A timestamp SHOULD* include the time zone.
-* If the timestamp includes a time zone, the LRS MAY be return the timestamp using a different timezone to the one originally used in the statement so long as the point in time referenced is not affected. The LRS SHOULD* return 
-the timestamp in UTC timezone. 
-* A timestamp MAY be truncated or rounded to a precision of at least 3 decimal digits for seconds (millisecond precision MUST be preserved). 
-* An LRS MUST NOT reject a timestamp for being from the future, to prevent issues due to clock errors.
+* A Timestamp MUST be formatted according to [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations).
+* A Timestamp MUST preserve precision to at least milliseconds (3 decimal points beyond seconds).  
+* A Timestamp SHOULD* include the time zone.
+* If the Timestamp includes a time zone, the LRS MAY be return the Timestamp using a different timezone to the one 
+* originally used in the statement so long as the point in time referenced is not affected. 
+* The LRS SHOULD* return the Timestamp in UTC timezone. 
+* A Timestamp MAY be truncated or rounded to a precision of at least 3 decimal digits for seconds. 
+* An LRS SHOULD* NOT reject a Timestamp for having a greater value than the current time, to prevent issues due to clock 
+* errors.
```

Comment:
Typo (new bullet). 

Also, based on the comments from @bscSCORM, this should be moved to the timestamp property. It doesn't make sense to apply to Stored.